artifacts/deploy/controller-manager.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         operator: Exists
       containers:
         - name: karmada-controller-manager
-          image: swr.ap-southeast-1.myhuaweicloud.com/karmada/karmada-controller-manager:latest
+          image: karmada-controller-manager:latest
```

Comment:
It does not work for 'hack/remote-up-karmada.sh'

Reply:
So if REGISTRY env is set in Makefile, we should add registry prefix to the image in all deploy artifacts ?

Reply:
At least, we should add registry prefix when 'hack/remote-up-karmada.sh' execute, otherwise applying artifacts/deploy/*.yaml to a standalone cluster(host cluster and not created by kind) will get a 'ErrImagePull' error

Reply:
ok